### ISSUE TITLE
Emails of people who come to our events

### ISSUE THREAD

**Issue by little-wow** [creativecommons/communications#17]
I'm opening an issue to discuss best practices of what to do with emails of people who come to events.

For people who come to an in person event, there are a few options that I can think of:
1. Have them sign a piece of paper with name and email and manually add to Civi
2. Set up a kiosk at the event to take names and emails
3. Send a post-event email that says "thanks for coming! Join our email list!" and point the to general signup page
4. Create an email signup landing page that's unique for events

I don't believe we can just import names from an event management system like Eventbrite without them opting into general emails from us, but we want to keep the metadata that they came to the event. Is there a way to import the list into Civi as opt out and then have them opt into further emails from us after we send the "thanks for coming" followup email? Is there precedence for this?

I think we'd like to know if a person came from an event rather than just off the Internet, and I'm not sure how to best handle that data. @mattl what do you think?

**Comment by rebeccakross** [comment]
Yes, we can import with a 'not opted in yet' (better than an opt-out). But first I'm curious to know if there is a 100% definitive way to know that we cannot add event attendees to our mailing list? Like a donation, attending an event seems like an expression of an interest in engaging. Just anecdotal, but I definitely receive emails after attending events, and find it strange/missed opp on the part of the org when I don't. Thoughts?

**Comment by tvol** [comment]
Yeah, i don't think there's any legal problem in doing it (at least in the United States). But it might be nice to indicate somewhere at the signup that we'll add them to our list.

**Comment by little-wow** [comment]
I am pretty sure that we can't unless they specifically say "Add me to your mailing list." We can, however, send them an email that says something like "Want more CC news? Sign up!"

**Comment by little-wow** [comment]
from Copyblogger's CANSPAM post:

"You absolutely must not:

Add anyone to your list without permission. For example, if you meet people at conferences and exchange business cards with them, don’t add those folks to your list. You also shouldn’t add previous business contacts or colleagues, even if you personally email them on a regular basis. When in doubt, always err on the side of caution, and never add people to your mailing list if you haven’t expressly received their permission to do so. (Editor’s note: We strongly recommended not adding anyone to your list without permission, but it’s not strictly speaking a CAN-SPAM requirement.)"

Basically, not strictly a CANSPAM issue, but I'd err on the side of double opt-in on this one as a best practice going forward.  Mailchimp, for one, makes you provide documentation as to where the people physically signed up

Did we send a "Hey, thanks for coming" email after CC:Rewire to everyone who signed up? If not, we should, and also provide a link to the newsletter.

**Comment by mattl** [comment]
We can send a follow up. We cannot add them to our list.

**Comment by rebeccakross** [comment]
Yes, we did send a follow up. I think an opt in at time of ticket registration is our best bet. Will do that going forward.

**Comment by little-wow** [comment]
Great.  Still a metadata question-- do we have a "Where did the person sign up" field in Civi @mattl ?
